libs/contact/package.json

```diff
@@ -14,6 +14,9 @@
     "test": "ng test contact --watch=false --browsers=ChromeHeadless",
     "publish": "(cd ../../dist/contact; npm publish --access=public)"
   },
+  "devDependencies": {
```

Comment:
You need a peerDependency on the `@daffodil/driver` too.

Reply:
addressed in b84a1d943f378222f669d30fb8715392307091b4